db/tests/resources/filter_sort_create.sql

```diff
@@ -0,0 +1,131 @@
+CREATE TABLE "FilterSort" (
```

Comment:
It doesn't matter much for a testing table, but underscores are preferable to capital letters for denoting words in SQL names (since SQL is non-case-sensitive by default).